app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -463,6 +474,15 @@ public boolean onLongClick(final View v) {
             case R.id.detail_controls_download:
                 NavigationHelper.openDownloads(getActivity());
                 break;
+
+            case R.id.detail_uploader_root_layout:
+                if (TextUtils.isEmpty(currentInfo.getSubChannelUrl())) {
+                    Log.w(TAG,
```

Comment:
why is this a warning? Apart from that, this info should only be put in the log in debug apks